introduction.md

```diff
@@ -0,0 +1 @@
+"Someone needs to put error codes for this project"
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.